### PR TITLE
formula for rem https://github.com/kykim/rem

### PR DESCRIPTION
**Pull request by kykim**

### CODE REVIEW

File: Library/Formula/rem.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Rem < Formula
+  desc "Command-line tool to access OSX Reminders.app database"
+  homepage "https://github.com/kykim/rem"
+  url "https://github.com/kykim/rem/archive/20150618.tar.gz"
+  # version "20150618"
```
Comment (bfontaine) [inline Library/Formula/rem.rb]: Please remove this comment.

File: Library/Formula/rem.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+               "-target", "rem",
+               "SYMROOT=build",
+               "DSTROOT=#{prefix}",
+               "INSTALL_PATH=/bin",
+               "-verbose",
+               "install"
+  end
+
+  test do
+    system "true"
```
Comment (bfontaine) [inline Library/Formula/rem.rb]: Could you please add a test for `rem`?

File: Library/Formula/rem.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+class Rem < Formula
+  desc "Command-line tool to access OSX Reminders.app database"
+  homepage "https://github.com/kykim/rem"
+  url "https://github.com/kykim/rem/archive/20150618.tar.gz"
+  # version "20150618"
+  sha256 "e57173a26d2071692d72f3374e36444ad0b294c1284e3b28706ff3dbe38ce8af"
+
+  depends_on :macos => :mountain_lion
+  depends_on :xcode
```
Comment (DomT4) [inline Library/Formula/rem.rb]: Is this runtime as well as for compile? i.e. if I install this, and then remove Xcode, will it work or not?

Comment (kykim) [inline Library/Formula/rem.rb]: If you install, then remove Xcode, it will work.

Comment (DomT4) [inline Library/Formula/rem.rb]: Thanks! In that case can you make the dependency `depends_on :xcode => :build`? Thanks!

**Comment by DomT4** [comment]
Ping.

**Comment by DomT4** [comment]
Presuming dead from lack of response. Please feel free to fix-up various comments and resubmit the PR if you're still interested, but closing here for now. Thanks for the submission!

**Pull request closed without merging**